docs/release-checklist.md

```diff
@@ -11,51 +11,26 @@ Martha Release Checklist
-        ```
-       gcloud auth login <your test account email>
-       curl -X POST https://us-central1-broad-dsde-staging.cloudfunctions.net/martha_v3 -H "Authorization: Bearer $(gcloud auth print-access-token)" -d '{"url": "drs://dg.712C/fa640b0e-9779-452f-99a6-16d833d15bd0"}' -H "Content-Type: application/json" | jq
-        ```
-    1. You should get a JSON object in response with the following structure:
-        ```js
-        {
-          "contentType": "...",
-          ...
-          "googleServiceAccount": {
-            ...
-          }
-        }
-        ```
-- [ ] Deploy and test on prod (including manual test)
+    1. [Run the full manual test on the dev environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng). The linked document instructs you to log into external servers via your profile page. For dev and other non-prod instances:
+        * For NIH Account, enter any username you want in the fake login page, then click on the generated link to return to your profile.
+        * For the other sites, log in with RAS account `Broadtestuser115`. Get the password from someone on the team if you're doing this for the first time.
+- [ ] Deploy and run manual test on **ALPHA**
+    1. Build on [Jenkins](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) as described in dev section above, setting TARGET to "alpha". 
+    1. [Run the full manual test on the alpha environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng) (see notes on account linking in dev environment above).
+- [ ] Deploy and run manual test on **STAGING**
+    1. Build on [Jenkins](https://fc-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) as described in dev section above, setting TARGET to "staging". 
+    1. [Run the full manual test on the alpha environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng) (see notes on account linking in dev environment above).
+- [ ] Deploy and run manual test on **PROD**
      1. Go to the [Jenkins prod manual deploy project](https://fcprod-jenkins.dsp-techops.broadinstitute.org/job/martha-manual-deploy/) and click on the "Build with Parameters" link.  For the parameters, you want to select the tag you just created (you shouldn't have a choice for TARGET).  Do not proceed until the job ends successfully.
          - You will need to be on the Broad network in order to have access.
-     1. [Run the full manual test on the prod environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng)
+     1. [Run the full manual test on the prod environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng). Use your real eRA Commons account to link with external services on your profile page.
```

Comment:
TOL: Perhaps a little more emphasis on using the "real" eRA Commons account here? It would be easy to assume this fourth iteration of the process was just like the preceding three and this is hiding out as the last sentence of the instructions. 🙂